src/org/usfirst/frc/team467/robot/ElevatorHeightSensor.java

```diff
@@ -37,20 +34,19 @@ public static ElevatorHeightSensor getInstance() {
 	}
 	
 	public boolean atMaxHeight() {
-		if(height > 8) {
+		if(getHeightFeet() > 8) {
 			return true;
 		}
 		
 		else return false;
 	}
 	
-	public double getHeight() {
-		height = rotations/feetConvert;
-		return height;
+	public double getHeightFeet() {
+		return rotations/ticksPerFoot;
```

Comment:
You can't just use "rotations" here since you don't set it anymore. Instead, you need to call getRotations(). I think you should remove the variable "rotations" from the class entirely.